src/client/components/About/About.css

```diff
@@ -0,0 +1,46 @@
+@import url("https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap");
+.about-section {
+  display: grid;
+  place-items: center;
+}
+.about {
```

Comment:
Think about the names you give to classes. You have to have the same standard as with function and component names.
Make them descriptive and unique. Then load your global classes in one css file